src/client/api/pythonApi.ts

```diff
@@ -247,15 +246,6 @@ export class PythonInstaller implements IPythonInstaller {
             this.interpreterPackages.trackPackages(resource);
         }
         let action: 'installed' | 'failed' | 'disabled' | 'ignored' = 'installed';
-        if (env.remoteName && reInstallAndUpdate) {
-            // Temporary work around for https://github.com/microsoft/vscode-jupyter/issues/6896
-            traceWarning(
```

Comment:
Is the bug still active? Wondering how we make sure to fix this for real before next release.

Reply:
The bug is active & moved to Python repo, https://github.com/microsoft/vscode-python/issues/16851
We cannot add a work around at our end.

Reply:
Reverting as the work around doesn't fix it (i thought it did, but was wrong).